modules/virt-golden-images.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE

```

Comment:
I recommend that you write a rough draft in a Google docs so that Adam and I can comment on it.